fabtests/common/shared.c

```diff
@@ -1147,9 +1175,16 @@ int ft_join_mc(void)
 int ft_av_insert(struct fid_av *av, void *addr, size_t count, fi_addr_t *fi_addr,
 		uint64_t flags, void *context)
 {
+	char tmp_buf[FT_MAX_CTRL_MSG];
 	int ret;
 
-	ret = fi_av_insert(av, addr, count, fi_addr, flags, context);
+	ret = ft_hmem_copy_from(opts.iface, opts.device, tmp_buf, addr, FT_MAX_CTRL_MSG);
+	if (ret) {
+		FT_PRINTERR("ft_hmem_copy_from", ret);
+		return ret;
+	}
```

Comment:
It's unclear why this change is needed (yes, I get it, but a future reader probably won't).  Maybe we should force OOB address exchange if hmem is used.